index.html

```diff
@@ -18,16 +21,178 @@
+            </div>
+            <!-- end card3 -->
+            <!-- start card4 -->
+            <div class="project-cards project4">
+                <div class="pjimgs">
+                    <img src="./img/pj4.svg" alt="pj4"/>
+                </div>
+                <div class="pjcontent">
+                    <p class="pjname">Project name goes here</p>
+                    <ul>
+                        <li>HTML/CSS</li>
+                        <li>Ruby on Rails</li>
+                        <li>Javasccript</li>
+                    </ul>
+                    <button class="pjbtn"><a href="#"><span>See this Project</span> <img src="./img/union.svg" alt="rightarrow"/></a></button>
+                </div>
+            </div>
+            <!-- end card4 -->
+            <!-- start card5 -->
+            <div class="project-cards project5">
+                <div class="pjimgs">
+                    <img src="./img/pj5.svg" alt="pj5"/>
+                </div>
+                <div class="pjcontent">
+                    <p class="pjname">Project name goes here</p>
+                    <ul>
+                        <li>HTML/CSS</li>
+                        <li>Ruby on Rails</li>
+                        <li>Javasccript</li>
+                    </ul>
+                    <button class="pjbtn"><a href="#"><span>See this Project</span> <img src="./img/union.svg" alt="rightarrow"/></a></button>
+                </div>
+            </div>
+            <!-- end card5 -->
+            <!-- start card6 -->
+            <div class="project-cards project6">
+                <div class="pjimgs">
+                    <img src="./img/pj6.svg" alt="pj6"/>
+                </div>
+                <div class="pjcontent">
+                    <p class="pjname">Project name goes here</p>
+                    <ul>
+                        <li>HTML/CSS</li>
+                        <li>Ruby on Rails</li>
+                        <li>Javasccript</li>
+                    </ul>
+                    <button class="pjbtn"><a href="#"><span>See this Project</span> <img src="./img/union.svg" alt="rightarrow"/></a></button>
+                </div>
+            </div>
+            <!-- end card6 -->
+        </div>
+        <!-- end project card -->
+    </section>
+    <!-- End WORK SECTION -->
+
+    <!-- Start ABOUT SECTION -->
+    <section class="about">
+        <div class="aboutintro">
+            <!-- start title -->
+            <div class="title">
+                <h1>About</h1>
+            </div>
+            <!-- end title -->
+            <div class="bio">
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
+            </div>
+
+            <button type="button" class="cvbtn"><a href="#"><span>Get My CV</span><img src="./img/downloadicon.svg" alt="icon"/></a></button>
```

Comment:
One key feature of a great-looking website is that they utilize proper spacing between contents, as such I recommend adding proper spacing between this button and the content which follows.

Your current copy 👇👇👇

<img width="281" alt="no margin" src="https://user-images.githubusercontent.com/84930355/205229354-f0e77658-b8ff-4f85-a5de-f7eec61f1974.png">




Template design 👇👇👇


<img width="451" alt="margin" src="https://user-images.githubusercontent.com/84930355/205229413-93cc3b0d-b5bc-4124-9b83-1fd7c9ecc018.png">


Kindly add some `margin-bottom` property here.